src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -537,7 +537,7 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   private loadResponseStatusForQuestion(model: QuestionEditFormModel): void {
     this.feedbackSessionsService.hasResponsesForQuestion(model.feedbackQuestionId)
         .subscribe((resp: HasResponses) => {
-          model.isQuestionHasResponses = resp.hasResponses;
+          model.isQuestionHasResponses = resp.hasResponses[''];
```

Comment:
Can I check if indexing with the empty string is needed for all occurrences of `hasResponses` now?

Reply:
Yes; in fact, `''` should be an enum that refers to the placeholder key index for all general purpose `hasResponses` query. I guess it is currently a magic string for now which I will have to replace.